src/components/Navigation.js

```diff
@@ -0,0 +1 @@
+import
```

Comment:
This is just an empty file with an import? Add content or leave this for another branch

Reply:
Creating a new branch for Nav.

Reply:
This component should be removed from this branch if it is not being used in this one.